paymentservice/views.py

```diff
@@ -30,3 +32,16 @@ class OrderDetailView(generics.RetrieveAPIView):
     def get_queryset(self):
         customer = self.request.user.customer
         return Order.objects.filter(customer=customer)
+
+
+class CreatePaymentView(APIView):
+    serializer_class = PaymentDetailsSerializer
+    authentication_classes = [JWTAuthenticator]
+    permission_classes = [IsAuthenticated]
+
+    def post(self, request, *args, **kwargs):
+        serializer = PaymentDetailsSerializer(data=request.data)
+        if serializer.is_valid():
+            serializer.save()
+            return Response({"payment_created": True}, status=status.HTTP_201_CREATED)
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
I think `generics.CreateAPIView` is sufficient here, so we don't have to write `post` explicitly